Cargo.toml

```
@@ -8,10 +8,10 @@ exclude = ["/examples", "/webview-dist", "/webview-src", "node_modules"]
 license = "Apache-2.0 OR MIT"
 
 [dependencies]
-tauri = "1.0.0"
-serde = "1.0.137"
-serde_json = "1.0.81"
-thiserror = "1.0.31"
-log = "0.4.17"
+tauri = "1.0"
+serde = "1.0"
+serde_json = "1.0"
+thiserror = "1.0"
+log = "0.4"
```

```suggestion
log = "0.4.17"
```

Should we leave this as-is bc technically `log` is still 0.x?

---

it is fine to keep it as 0.4